.github/codecov.yml

```
@@ -0,0 +1,26 @@
+# yaml-language-server: $schema=https://json.schemastore.org/codecov.json
+
+# https://docs.codecov.com/docs/codecov-yaml
+
+coverage:
+  status:
+    project: # https://docs.codecov.com/docs/commit-status
```

```suggestion
    patch: off
    project: # https://docs.codecov.com/docs/commit-status
```

IMHO Patch coverage is more of a burden. i.e. if you edit a file that was untested previously, it'll fail the check.